tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -169,20 +169,22 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
     sqs_body["provider"] = {}
     sqs_body["collection"] = {}
     sqs_body["granule"] = {}
+    sqs_body["provider"][
+        "providerName"
+    ] = None  # set to None because cumulus only returns providerId for now. In case it is available in the future, update orca_copy_to_glacier_workflow.asl.json and config.json as needed
```

Comment:
So this is a really long line. May want to think of putting the comment above `sqs_body["provider"]["providerName"] = None` as a multi line comment for better readability.

Reply:
updated